builder/js/ncf.js

```diff
@@ -847,23 +849,22 @@ $scope.groupMethodsByCategory = function () {
         $scope.techniques[index] = savedTechnique;
       }
     }
-    var saveError = function(action, data) {
-      $scope.saving = false;
+    $scope.saveError = function(action, data) {
```

Comment:
Can you switch back to a var here ? or a function directly !! 

would be simpler !!